docs/source/installation.rst

```diff
@@ -214,6 +219,24 @@ your environment is activated and linked to whatever IDE you are using.  If you
 Notebooks, follow `these instructions <https://janakiev.com/blog/jupyter-virtual-envs/>`_ for
 adding your virtual environment as a new kernel for your notebook.
 
+Dependency error on mac ARM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are using a mac with an ARM processor, you may encounter an error when installing
+``sktime[all_extras]``.  This is due to the fact that some libraries included in ``all_extras``
+are not prepared to run in ARM-based processors.
+
+If you want to still install all the remaining compatible libraries from ``all_extras`` you need
+to take into account the following:
+
+* Avoid the following packages:
```

Comment:
would make it clearer: *do not install*, since "avoid" is a vague verb.

General advice for writing tech docs, think whether it is clear to the reader: How does it translate into action? look away from the computer screem while these are being installed ? :-)